elyra/pipeline/resources/palette.json

```diff
@@ -53,7 +53,7 @@
       ]
     },
     {
-      "label": "Python Script",
+      "label": "Python",
       "image": "",
       "id": "python-script",
       "description": "Python Script",
```

Comment:
If this is the component description in the palette, it should probably read "Run Python script" because a node implements a task, not a file of that type.

Reply:
@kiersten-stokes why we change these? Could we leave it the way it is on master?